.circleci/config.yml

```diff
@@ -183,22 +163,19 @@ commands:
     parameters:
       rust-version:
         type: string
-        default: << pipeline.parameters.stable-rust-version >>
+        default: ""
     steps:
-      - test-setup:
-          rust-version: <<parameters.rust-version>>
+      - test-setup
       # Test with 1. only default features on, 2. all features on, 3. no features on.
       # This is not perfect (really we want the cartesian product), but is good enough in practice.
       - run:
           name: Test (default features, no features, and all features)
-          command: bash automation/all_rust_tests.sh --verbose
+          command: RUSTUP_TOOLCHAIN="<<parameters.rust-version>>" automation/all_rust_tests.sh --verbose
```

Comment:
Using an environment variable like this will override the `rust-toolchain` file.